install.php

```diff
@@ -133,7 +133,7 @@ function ArraySearchRecursive($Needle,$Haystack,$NeedleKey="",$Strict=false,$Pat
 	$usePeople=($result->rowCount()>0 && !ArraySearchRecursive('fac_People',$test))?($result->rowCount()>0 && !ArraySearchRecursive('fac_people',$test))?false:true:true;
 
 	// New install so create a user
-	require_once("customers.inc.php");
+	require_once("classes/People.class.php");
```

Comment:
They're both needed, actually, since it should/did do upgrades prior to the class splitting business.